aws/request/request.go

```diff
@@ -429,13 +433,20 @@ func (r *Request) getNextRequestBody() (io.ReadCloser, error) {
 // If no error occurs the length of the body will be returned.
 func computeBodyLength(r io.ReadSeeker) (int64, error) {
 	seekable := true
+
 	// Determine if the seeker is actually seekable. ReaderSeekerCloser
 	// hides the fact that a io.Readers might not actually be seekable.
 	switch v := r.(type) {
 	case aws.ReaderSeekerCloser:
```

Comment:
The logic here is the same? Would it make sense to have a fallthrough for the `aws.ReadSeekerCloser` case?

Reply:
Maybe, I'll try that. The types are different so not sure what that will do with ah`fallthrough`